### PR TITLE
Updating activities for Yelle Lieder

### PR DESCRIPTION
**Pull request by LieneKal**
Hi there!
The following activity or activities have been submitted to the Green Software Champions directory with at least one new activity aligned to a project or working group you lead. By clicking "files changed" you can see the activities that have been submitted and the linked project.
Please could you now check these activities in "Files Changed" and then push the review changes button - then comment giving approval or asking for clarification.
If you have any questions, please comment to @LieneKal and @adamj89 or email champs-team@greensoftware.foundation.

### CODE REVIEW

File: src/content/champions/yelle-lieder.yaml [hunk 1]
```diff
@@ -15,6 +15,24 @@ social:
   github: yellelieder
   linkedin: lieder
 activities:
+  - contributionType: oss contribution
+    linkedGSFProject: Carbon Aware SDK
```
Comment (Willmish) [inline src/content/champions/yelle-lieder.yaml:19]: I have not seen this project before and I do not think it is directly linked to https://github.com/Green-Software-Foundation/carbon-aware-sdk . Please specify how was this a contribution to the Carbon Aware SDK, or any related pull requests.

**Review by Willmish — CHANGES_REQUESTED** [review]
See attached review comment - clarify the connection of the project to CA SDK.

File: src/content/champions/yelle-lieder.yaml (hunk 1 above)
Comment (LieneKal) [reply src/content/champions/yelle-lieder.yaml:19]: @adamj89 can you have a look at this, please?

**Review by adamj89 — APPROVED** [review]
Other than the comment by @Willmish (thank you) - no comments were recieved so we are assuming your approval. 

Please review the [Project Lead & WG Chairs Guide for Champions](https://grnsft.org/champions/chairguide) for reference on the expectation of Chairs & Leads in the Champion approval process.

**Pull request merged**